web/routes.js

```diff
@@ -6,10 +6,9 @@ import LandingPage from 'components/LandingPage';
 export default (
     <div className='routes-container'>
         <Switch>
-            <Route exact path='/' component={LandingPage}/>
-            <Route path='/dashboard' component={LandingPage}/>
-            <Route path='/teamlist' component={LandingPage}/>
+            <Route exact path='/' component={LandingPage} />
+            <Route path='/dashboard' component={LandingPage} />
+            <Route path='/teamlist' component={LandingPage} />
```

Comment:
Let's change `/teamlist` to simply `/team`.

Also as part of #4 let's consider what routing options we will need here based off some backend assumptions yet to be developed... 

- Each team member will have an ID so we could have a route like `/team/member/:teamMemberId` that shows that team member's schedule for the week
- We need a route + component for scheduling your own Setblocks, and maybe even one for editing a specific Setblock. Each setblock will have it's own ID on the backend as well so something like `/schedule/edit/:setblockId`

Preliminary list of routes for #4:
- `/` - LandingPage
- `/team` - TeamPage
- `/team/:teamMemberId` - TeamMemberPage
- `/schedule` - SchedulePage, with today's day selected by default
- `/schedule/:dayOfWeek` - SchedulePage, with correct day selected. `dayOfWeek` will be an integer from 0 to 6 (0 = Monday, 6 = Sunday)
- `/schedule/edit/:setblockId` - SetblockDetailPage